docs/source/technical_overview.rst

```diff
@@ -65,13 +65,13 @@ operating on objects of the following types:
     .. |dimension|        replace:: :attr:`dimension <pymor.vectorarrays.interface.VectorArray.dim>`
     .. |empty|            replace:: :meth:`~pymor.vectorarrays.interface.VectorSpace.empty`
     .. |id|               replace:: :meth:`~pymor.vectorarrays.interface.VectorSpace.id`
-    .. |indexed|          replace:: :meth:`indexed <pymor.vectorarrays.interface.VectorArray.__getitem__>`
+    .. |indexed|          replace:: :meth:`!indexed <pymor.vectorarrays.interface.VectorArray.__getitem__>`
```

Comment:
What's the issue here? This actually has a docstring?

Reply:
I guess magic methods don't appear in the documentation.